docs/modules/ROOT/pages/configurations.adoc

```diff
@@ -16,6 +16,11 @@ The tools and configurations below are required to run aiSSEMBLE.
 * Continuous Deployment: https://docs.tilt.dev/install.html[Tilt,role=external,window=_blank] for local development
 * Source Code Management: https://git-scm.com/[git,role=external,window=_blank]
 
+==== Maven Build Cache
+aiSSEMBLE supports use of the Maven Build Cache to accelerate your project's development.  Newly created aiSSEMBLE
+based projects are automatically configured to use the build cache.  To add this capability to an existing project,
+refer to the xref:https://maven.apache.org/extensions/maven-build-cache-extension/[official Maven documentation], as well as the xref:https://github.com/boozallen/aissemble/tree/dev/foundation/foundation-archetype/src/main/resources/archetype-resources/.mvn[sample configurations] provided by the aiSSEMBLE archetype.
```

Comment:
S: Links that navigate away from the GH pages docs should follow our [external link guidelines](https://github.com/boozallen/aissemble/blob/dev/docs/CONTRIBUTING.md#markup-standards)

Reply:
Done.